lib/toolkit/loader.js

```diff
@@ -785,7 +786,20 @@ function Loader(options) {
     if (isNative && !uri)
       uri = id;
     let module = Module(id, uri);
-    module.exports = freeze(modules[id]);
+
+    // Lazily expose built-in modules in order to
+    // allow them to be loaded lazily.
+    let cachedModules = modules;
+    Object.defineProperty(module, "exports", {
+      enumerable: true,
+      get: function() {
+        return cachedModules[id];
+      },
+      set: function(v) {
```

Comment:
Oh right, I completely mixed up things, I original defined a setter in order to ensure that `exports` behavior is the same than this one defined over here:
https://github.com/ochameau/addon-sdk/blob/lazyModules/lib/toolkit/loader.js#L699-L700
```
 exports: { enumerable: true, writable: true, value: create(null),
configurable: true },
```
But here, we are past that, now exports is meant to be frozen on Module instance... right ?
I removed the setter.